common/main/en_SL.xml

```diff
@@ -40,6 +40,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 	<numbers>
 		<currencies>
 			<currency type="SLL">
+				<symbol draft="contributed">Le (1964-2022)</symbol>
```

Comment:
Should we also remove this symbol like the others?

Reply:
thanks, thought I caught them all

Reply:
forgot to save that file.